jis-documentation/src/filters.adoc

```diff
@@ -251,4 +251,42 @@ include::{sourcedir}/filters/SobelFilterTest.java[tags=edgeDetectionSobelFilter]
 ----
 --
 image::Sobel.png[title="Sobel Filter", width="300", height="300"]
+
```

Comment:
Documentation doesn't work properly. You replaced "--" after  image::Sobel.png[title="Sobel Filter", width="300", height="300"]

Also you tags don't work, should be LuminanceChanelFilterTest.java[tags=LuminanceChanelFilterTest] etc